wemake_python_styleguide/visitors/tokenize/syntax.py

```diff
@@ -36,6 +37,16 @@ def visit_dot(self, token: tokenize.TokenInfo) -> None:
         """
         self._check_line_starts_with_dot(token)
 
+    def visit_newline(self, token: tokenize.TokenInfo) -> None:
+        r"""
+        Checks \r (carriage return) in line breaks.
```

Comment:
We need to wrap `\r` into double backquotes

Reply:
fixed